test/support/helpers/review-charge-element-result.helper.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+/**
+ * @module ReviewChargeElementResultHelper
+ */
+
+const { generateUUID } = require('../../../app/lib/general.lib.js')
+const ReviewChargeElementResultModel = require('../../../app/models/review-charge-element-result.model.js')
+
+/**
+ * Add a new charge element result for 2pt matching
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `id` - [random UUID]
+ * - `chargeElementId` - [random UUID]
+ * - allocated - 0
+ * - aggregate - 1
+ * - chargeDatesOverlap - false
+ * - updatedAt - new Date()
```

Comment:
The `created_at` and `updated_at` get automatically populated when the record is created. So, we shouldn't need to default them.